cli/src/generate/parse_grammar.rs

```diff
@@ -119,7 +119,10 @@ pub(crate) fn parse_grammar(input: &str) -> Result<InputGrammar> {
         precedence_orderings.push(ordering);
     }
 
-    let extra_symbols = grammar_json.extras.into_iter().map(parse_rule).collect();
+    let extra_symbols = grammar_json.extras.into_iter()
+                                           .map(parse_rule)
+                                           .filter(|rule| !Rule::is_empty_string(rule))
```

Comment:
This still allows a `NamedSymbol` which has an empty string inside. Should we filter that as well?